codediff

```diff
@@ -97,4 +96,4 @@ if __name__ == '__main__':
     except (FileError, FileIOError) as e:
         argparser.error(str(e))
     except Exception as e:
-        _logger.exception('Oh noes! :( An exception has occured.')
+        _logger.exception('Oh noes! :( An exception has occured.')
```

Comment:
"occurred"?

Reply:
Ah yes, that doesn't seem to hard to fix. My mistake!